src/components/BookmarkCard/BookmarkCardIcon/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import './index.scss';
+
+type BookmarkCardIconProps = {
+  className?: string;
+  black?: boolean;
+  size: 'sm' | 'md' | 'lg';
+};
+
+const BookmarkCardIcon = ({
+  size,
+  black,
+  className
+}: BookmarkCardIconProps) => {
+  const classes = classnames(
+    'fa',
+    'fa-bookmark',
+    'bookmark__icon',
+    {
+      'fa-1x': size === 'sm'
+    },
+    {
+      'fa-2x': size === 'md'
+    },
+    {
+      'fa-3x': size === 'lg'
+    },
+    {
+      black
+    },
+    className
+  );
+
+  return (
+    <i className={classes} data-testid="bookmark-icon" aria-hidden="true" />
```

Comment:
I think the intent of this icon is that it will be eventually clickable. I suppose it makes sense to keep this an `<i>`, but maybe we can add a hover to set `cursor: pointer;`?

Reply:
Yep definitely.  The integration/interaction to the systemList table and the db is in another story, but makes sense to have that UI a part of this story.